build_sysext

```diff
@@ -186,6 +218,17 @@ if [[ -n "${FLAGS_manglefs_script}" ]]; then
   "${FLAGS_manglefs_script}" "${BUILD_DIR}/install-root"
 fi
 
+if [[ "$FLAGS_generate_pkginfo" = "${FLAGS_TRUE}" ]] ; then
+  info "  Creating pkginfo squashfs '${BUILD_DIR}/${SYSEXTNAME}_pkginfo.raw'"
+  mkdir -p "${BUILD_DIR}/img-pkginfo/var/db"
+  cp -R "${BUILD_DIR}/install-root/var/db/pkg" "${BUILD_DIR}/img-pkginfo/var/db/"
+  mksquashfs "${BUILD_DIR}/img-pkginfo" "${BUILD_DIR}/${SYSEXTNAME}_pkginfo.raw" -noappend
+fi
+
+info "Writing ${SYSEXTNAME}_packages.txt"
+ROOT="${BUILD_DIR}/install-root" PORTAGE_CONFIGROOT="${BUILD_DIR}/install-root"\
+      equery --no-color list --format '$cpv::$repo' '*' > "${BUILD_DIR}/${SYSEXTNAME}_packages.txt"
+
```

Comment:
I think that this code should run before the manglefs script is invoked as it could easily just do `rm -rf "${rootfs}/var"`, resulting in no database and no packages listing.

Reply:
That would be ... _evil_! Good catch, will move it.